### ISSUE TITLE
FoodBIMS

### ISSUE THREAD

**Issue by Mailaender** [OpenChrom/OpenTyper#31]
http://bioinformatica.isa.cnr.it/Bact_Dbase.htm looks like it was discontinued early on and the library remains small. Data is only provided in HTML tables under standard Copyright terms.

**Comment by eselmeister** [comment]
@Mailaender I would recommend to get in contact with the group. Ask the for permission to use, extend and distribute the data.

**Comment by Mailaender** [comment]
Good idea. Contacted the group via the provided webform. http://bioinformatica.isa.cnr.it/form_per_info.htm
